### PR TITLE
feat(ci): add deployment to preview site in CI/CD

### PR DESCRIPTION
**Pull request by lewisdaly**
closes mojaloop/project#2282

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  version: 2
+  build_and_test:
+    jobs:
+      - build_and_deploy_s3:
+          context: org-global
+          filters:
+            branches:
+              only:
+                # TODO: change this to `mojaloop-docs-2.0`
+                - /feat\/2282-.*/
```
Comment (meekg33k) [inline .circleci/config.yml:85]: This should be pointing to the `mojaloop-docs-2.0` branch correct?

**Review by meekg33k — APPROVED** [review]
One minor comment, otherwise lgtm!

File: .circleci/config.yml (hunk 1 above)
Comment (lewisdaly) [reply .circleci/config.yml:85]: Ah yeah. I'll update it now

**Review by meekg33k — APPROVED** [review]

**Comment by lewisdaly** [comment]
Thanks @meekg33k

**Pull request merged**